src/utilities/random.ts

```diff
@@ -1,9 +1,11 @@
+type RandomRollFnc = () => 4
+
 const RANDOM_NUMBER = 4
```

Comment:
I hate you.

Reply:
Hey I could've made it worse by making a random number type